app/filters.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Instantiate object used to store the methods registered as a
+ * 'filter' (of the same name) within nunjucks. You can override
+ * gov.uk core filters by creating filter methods of the same name.
+ * @type {Object}
+ */
+var filter = {};
```

Comment:
@joelanman I debated that one. You're probably write. Just felt easier to read as the singular when composing them but that could just be me.